broker/src/main/resources/management-schema.xml

```diff
@@ -39,11 +39,11 @@
   </sbe:message>
 
   <sbe:message name="SnapshotChunk" id="4">
-    <field name="snapshotPosition" id="0" type="uint64"/>
     <field name="totalCount" id="1" type="int32"/>
     <field name="checksum" id="2" type="uint64"/>
     <data name="chunkName" id="3" type="varDataEncoding"/>
     <data name="content" id="4" type="blob"/>
+    <data name="snapshotId" id="5" type="varDataEncoding" />
```

Comment:
Just rename the position, should be fine.

Reply:
Does it matter? What's the difference?

Reply:
The difference is that you currently use 5 instead of 0 so we are not using index 0 :sweat_smile:

Reply:
I figured best practice was to always add fields, but I guess I'm not sure what happens if you remove one :sweat_smile: 

Either way, it doesn't matter much since we're already breaking everything I guess :man_shrugging: